### ISSUE TITLE
[APPRECIATION] Congratulations!

### ISSUE THREAD

**Issue by angelhdzmultimedia** [Maiquu/nuxt-quasar#56]
### Describe the feature

Just saw the module featuring in the official Nuxt page!!! https://nuxt.com/modules/quasar

### Congratulations!

I'm building a CLI tool to create Nuxt apps and scaffolding, look who's part of the dependencies!

![image](https://github.com/Maiquu/nuxt-quasar/assets/10191085/b34ff2c1-4b97-488f-80dd-23233bd9b5c0)
![image](https://github.com/Maiquu/nuxt-quasar/assets/10191085/155926e3-a0bc-4eb6-8795-b250f17e19e0)
![image](https://github.com/Maiquu/nuxt-quasar/assets/10191085/6ff12dd3-9d67-436c-ab0e-a76652926597)

I will keep up to date to any news regarding the flicker with dark mode using Quasar in SSR mode.
For the meantime I keep using `ssr: false` (SPA).

**Wishing you health, peace, and success.**

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Does this feature have a potential to break existing features?

**Comment by Maiquu** [comment]
Much appreciated!

> I will keep up to date to any news regarding the flicker with dark mode using Quasar in SSR mode.

Does the issue persist even with the solution provided [here](https://github.com/Maiquu/nuxt-quasar/issues/16#issuecomment-1481817962)?

**Comment by angelhdzmultimedia** [comment]
> Much appreciated!
> 
> > I will keep up to date to any news regarding the flicker with dark mode using Quasar in SSR mode.
> 
> Does the issue persist even with the solution provided [here](https://github.com/Maiquu/nuxt-quasar/issues/16#issuecomment-1481817962)?

Sorry for the confusion. The solution provided is to use nuxt cookies to store and retrieve the Dark mode value.

The issue I'm referring to (I forgot to give you context haha 😅) is this https://github.com/piscis/nuxt-quasar-boilerplate/issues/31

It has to do with an ongoing issue with Quasar and SSR that the creator is yet to solve, to avoid the flickering when applying the Dark mode at runtime. It has to do with components hydration, I don't know.

In the link I provided, there's a video I recording where it shows the flickering I'm referring to.

It stays in light (default) more 1-2 seconds, and then it changes to Dark Mode. I tried so many things to solve it, 
even setting the Dark mode in the server side.
